pom.xml

```diff
@@ -136,17 +136,4 @@
     </pluginRepository>
   </pluginRepositories>
 
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-deploy-plugin</artifactId>
-        <inherited>false</inherited>
-        <configuration>
-          <skip>true</skip>
```

Comment:
Useless to not skip, since the module POMs will be flattened, but harmless enough to publish.